proto/ibc/core/client/v1/query.proto

```diff
@@ -51,7 +51,14 @@ service Query {
   rpc UpgradedClientState(QueryUpgradedClientStateRequest)
       returns (QueryUpgradedClientStateResponse) {
     option (google.api.http).get =
-        "/ibc/core/client/v1/upgraded_client_states/{client_id}";
+        "/ibc/core/client/v1/upgraded_client_states";
```

Comment:
```suggestion
        "/ibc/core/client/v1/upgraded_client_state";
```

There is only one upgraded client state at a time

Reply:
this is an API thing. I think it needs to be plural